sunpy/map/mapbase.py

```diff
@@ -1030,35 +1031,12 @@ def rotate(self, angle=None, rmatrix=None, order=4, scale=1.0,
         to rotation, and differences from IDL's rot().
         """
         if angle is not None and rmatrix is not None:
-            raise ValueError("You cannot specify both an angle and a matrix")
+            raise ValueError("You cannot specify both an angle and a rotation matrix.")
         elif angle is None and rmatrix is None:
             rmatrix = self.rotation_matrix
 
-        # This is out of the quantity_input decorator. To allow the angle=None
-        # case. See https://github.com/astropy/astropy/issues/3734
-        if angle:
```

Comment:
For posterity, this is the line where the truth value of an array was being calculated